packages/mwp-app-render/src/components/uxcapture/UXCapture.jsx

```diff
@@ -15,21 +15,16 @@ export const onMark = (mark: string) => {
 		return;
 	}
 
-	const markStartTime = window.performance
+	const performanceMark = window.performance
 		.getEntriesByType('mark')
-		.find(entry => entry.name === mark).startTime;
+		.find(entry => entry.name === mark);
 
 	// Set a marker in the trace details
 	window.newrelic.addToTrace({
 		name: mark,
-		start: markStartTime,
+		start: performanceMark.startTime, // this is an epoch ms timestamp
```

Comment:
My bad - I was reading the docs about `mark.startTime` and saw that it had a `timestamp` value type, but apparently that doesn't mean it's a ms-since-epoch 'timestamp'. The performance spec uses [this definition of 'time origin'](https://www.w3.org/TR/hr-time-2/#dfn-time-origin) to determine the start time, and I guess `navigationStart` is [almost the same thing](https://www.w3.org/TR/navigation-timing/#performancetiming), so should be fine for our needs.

Will follow up

Reply:
Thanks for the link, I was trying to find specific reference, because all those "timestamp" references were very ambiguous, I guess I should've read it more thoroughly myself ;)